### DISCUSSION TITLE
Contribution Experience

### DISCUSSION THREAD

**Discussion by adityathakurxd** [adityathakurxd/university#32] [Show and tell]
### Thank you so much for your contribution! 
We'd love to build and ship this project so it can be used by students across universities.

**Please do star the repository** 🌟 and, if possible share your story to LinkedIn and feel free to tag me. (@adityathakurxd)

## Please share your experience working on this repository! 👇

- Was it easy to set up?
- What did you learn working on the project?
- Was there anything that can be done better?
- Feature suggestion and feedback on idea.

**Comment by 3kdeveloper** [comment]
Can I work on Profile screen ???

**Reply by adityathakurxd** [reply]
Sure @3kdeveloper I'll create an issue and assign it to you.
